src/Sulu/Bundle/DocumentManagerBundle/Initializer/Initializer.php

```diff
@@ -59,20 +53,14 @@ public function initialize(OutputInterface $output = null, $purge = false)
     {
         $output = $output ?: new NullOutput();
 
-        if (true === $purge) {
-            $output->write('!! Purging workspaces');
-            $this->purger->purge();
-            $output->writeln(' [OK]');
-        }
-
         arsort($this->initializerMap);
 
         foreach (array_keys($this->initializerMap) as $initializerId) {
             $output->writeln(sprintf('<comment>%s</>', $initializerId));
             $initializer = $this->container->get($initializerId);
-            $initializer->initialize($output);
+            $initializer->initialize($output, $purge);
         }
         $output->write(PHP_EOL);
-        $output->writeln('<comment>*</> Legend: [+] Added [*] Updated [ ] No change');
+        $output->writeln('<comment>*</> Legend: [+] Added [*] Updated [-] Purged [ ] No change');
```

Comment:
@chirimoya The other options here have been explained, and when thinking about it `-` makes more sense for purging, also adapted the `RootPathPurgeInitializer`.